Assets/Scripts/KatamariMovement.cs

```diff
@@ -353,9 +360,10 @@ public void UpdateTimeStateMachine()
                 hitXAngle = 45;
 
                 // Set up physics
-                forceMode = ForceMode.Impulse;
                 rb.constraints = RigidbodyConstraints.FreezePosition;
                 rb.freezeRotation = false;
+                Time.timeScale = 1.0f;
+                Time.fixedDeltaTime = Time.timeScale * 0.02f;
```

Comment:
Just in case these get out of whack when transitioning back to golf mode.

Reply:
yaaa, I may have fixed that in my branch, i will get that shit pushed as soon as i can